packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/command/command.controller.js

```diff
@@ -0,0 +1,11 @@
+export default class CommandCtrl {
+  $onInit() {
+    if (!this.data) {
+      this.goBack(true);
+    }
```

Comment:
**perf:** `redirectTo` in state definition is the right place to handle this scenario as the state will not be loaded if `data` is not present.

Reply:
Good point :) Fixed !